src/parser/jobs/whm/modules/Thinair.tsx

```diff
@@ -106,17 +122,15 @@ export class Thinair extends Analyser {
 		}
 
 		const panels = this.history.map(record => {
-			const actions = record.casts.map(action => getDataBy(ACTIONS, 'id', action)).filter(x => !!x)
-			const gcds = actions.filter(action => action && action.onGcd)
-			const numGcds = gcds.length
-
 			return {
 				key: record.start,
 				title: {
 					content: <Fragment>
```

Comment:
Since the "root" element is in a `<span>` tag, consider removing the `<Fragment>` tag for this.

Reply:
Tag has been removed.